### PR TITLE
Donation Form (7)

### PR DESCRIPTION
**Pull request by meganniu**
[Staging](https://bp-pregnancy-centre-staging.web.app/donation-form)
[Figma](https://www.figma.com/file/4EFYFvPUgv1aBdzkE12xn3/Arranging-a-Donation-(Phase-2)?node-id=531%3A150)
Design Specs:
- https://www.notion.so/uwblueprintexecs/72616a24aa4e482b84bf683f919eae0f?v=378202a223194dbcbf8b60847a3f9618&p=31555f0f9a2940818f16bbecfab93738
- https://www.notion.so/uwblueprintexecs/72616a24aa4e482b84bf683f919eae0f?v=378202a223194dbcbf8b60847a3f9618&p=ea44a0bc7c3543bf9dffa033344d5b36
- https://www.notion.so/uwblueprintexecs/72616a24aa4e482b84bf683f919eae0f?v=378202a223194dbcbf8b60847a3f9618&p=a83c29169841407dace84b08de505d02

See the component at `http://localhost:3000/test` 

https://user-images.githubusercontent.com/20423590/124369330-b9e4f000-dc38-11eb-81cf-6b9891b65b6b.mov

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -79,13 +79,13 @@ workflows:
...
     jobs:
       - lint-server
-      - deploy-server:
-          context:
-            - tpc-staging
-          filters:
-            branches:
-              only:
-                - staging
+      # - deploy-server:
```
Comment (meganniu) [inline .circleci/config.yml:82]: Disable deployment to staging server. Staging server is broken rn because of billing issues

**Review by Anand-Issac — APPROVED** [review]
Lgtm

**Review by kevin-wangg — APPROVED** [review]
LGTM!

**Comment by amandatdu** [comment]
When a donor first types in the Phone number or email field, they automatically get an error message like suggestion. This might scare them a little – they're not doing anything wrong yet ahha! Could we change the colour of the text & box to just dark gray and remove the error sign? It'll only be an error if they try move on without entering a proper phone number or email (by pressing next)!

**Comment by amandatdu** [comment]
If I move onto the Item details step, I should be able to move back to the previous step without needing to save an item if all of the fields are empty.

**Comment by amandatdu** [comment]
Super minor design change! Can we remove the horizontal line between the Item Detail Tags? The horizontal line can stay if there are one or more active Item Detail forms, but it shouldn't be there if there are no Item Details Forms.
Screen Shot 2021-07-12 at 9 30 42 PM

**Comment by amandatdu** [comment]
Hi! I completely forgot to mention another thing! The hover pop ups on the quantity and age "?" don't really follow whats designed. 

I have provided what the designs look like here: 

https://www.figma.com/file/4EFYFvPUgv1aBdzkE12xn3?node-id=518:3218#91424874
https://www.figma.com/file/4EFYFvPUgv1aBdzkE12xn3?node-id=518:3310#91457706

Thanks!

Screen Shot 2021-07-15 at 10 10 52 PM
Screen Shot 2021-07-15 at 10 10 47 PM

**Pull request merged**